packages/core/src/app/payment/paymentMethod/HostedCreditCardFieldsetValues.ts

```diff
@@ -12,15 +10,6 @@ export default interface HostedCreditCardFieldsetValues {
     };
 }
 
-export interface HostedCreditCardValidationValues extends CardInstrumentFieldsetValues {
-    hostedForm: {
-        errors?: {
-            cardCodeVerification?: string;
-            cardNumberVerification?: string;
-        };
-    };
-}
-
```

Comment:
Unused interface. It's been extracted to `packages/hosted-credit-card-integration/src/types/HostedCreditCardFiledsetValues.ts`.